"use strict";(self.webpackChunkcore=self.webpackChunkcore||[]).push([[6110],{5680:(e,t,n)=>{n.d(t,{xA:()=>d,yg:()=>m});var a=n(6540);function o(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function r(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function i(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?r(Object(n),!0).forEach((function(t){o(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,a,o=function(e,t){if(null==e)return{};var n,a,o={},r=Object.keys(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||(o[n]=e[n]);return o}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(o[n]=e[n])}return o}var l=a.createContext({}),c=function(e){var t=a.useContext(l),n=t;return e&&(n="function"==typeof e?e(t):i(i({},t),e)),n},d=function(e){var t=c(e.components);return a.createElement(l.Provider,{value:t},e.children)},p="mdxType",g={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},u=a.forwardRef((function(e,t){var n=e.components,o=e.mdxType,r=e.originalType,l=e.parentName,d=s(e,["components","mdxType","originalType","parentName"]),p=c(n),u=o,m=p["".concat(l,".").concat(u)]||p[u]||g[u]||r;return n?a.createElement(m,i(i({ref:t},d),{},{components:n})):a.createElement(m,i({ref:t},d))}));function m(e,t){var n=arguments,o=t&&t.mdxType;if("string"==typeof e||o){var r=n.length,i=new Array(r);i[0]=u;var s={};for(var l in t)hasOwnProperty.call(t,l)&&(s[l]=t[l]);s.originalType=e,s[p]="string"==typeof e?e:o,i[1]=s;for(var c=2;c<r;c++)i[c]=n[c];return a.createElement.apply(null,i)}return a.createElement.apply(null,n)}u.displayName="MDXCreateElement"},9084:(e,t,n)=>{n.r(t),n.d(t,{assets:()=>l,contentTitle:()=>i,default:()=>g,frontMatter:()=>r,metadata:()=>s,toc:()=>c});var a=n(8168),o=(n(6540),n(5680));const r={},i="The Genome in a Bottle Consortium (NIST)",s={unversionedId:"Public-Datasets/NIST",id:"Public-Datasets/NIST",title:"The Genome in a Bottle Consortium (NIST)",description:"We have collected the NIST Ashkenazim (HG003, HG004, HG002) Jewish data set for public use and comparison testing.",source:"@site/docs/Public-Datasets/NIST.md",sourceDirName:"Public-Datasets",slug:"/Public-Datasets/NIST",permalink:"/Core-Docs/docs/Public-Datasets/NIST",draft:!1,editUrl:"https://github.com/UCGD/Core-Docs/docs/docs/Public-Datasets/NIST.md",tags:[],version:"current",frontMatter:{},sidebar:"tutorialSidebar",previous:{title:"1000 Genomes Phase 3 WGS",permalink:"/Core-Docs/docs/Public-Datasets/1000G"},next:{title:"Introduction",permalink:"/Core-Docs/docs/UCGD-Pipeline/introduction"}},l={},c=[{value:"Quick data access:",id:"quick-data-access",level:4},{value:"Data collection.",id:"data-collection",level:2},{value:"Important commands:",id:"important-commands",level:3},{value:"Project setup",id:"project-setup",level:2},{value:"Comparison of version 2.0.0+ of the UCGD-Pipeline",id:"comparison-of-version-200-of-the-ucgd-pipeline",level:2},{value:"Downloaded benchmark files:",id:"downloaded-benchmark-files",level:3},{value:"Isolated HG002 Individual",id:"isolated-hg002-individual",level:3},{value:"Running RTG",id:"running-rtg",level:3},{value:"Results:",id:"results",level:3},{value:"ROC",id:"roc",level:4},{value:"Precision/Sensitivity",id:"precisionsensitivity",level:4},{value:"Squash-ploidy-pass summary:",id:"squash-ploidy-pass-summary",level:4},{value:"Review of False Negative Calls:",id:"review-of-false-negative-calls",level:4},{value:"Sentieon DNAscope comparison.",id:"sentieon-dnascope-comparison",level:2}],d={toc:c},p="wrapper";function g(e){let{components:t,...r}=e;return(0,o.yg)(p,(0,a.A)({},d,r,{components:t,mdxType:"MDXLayout"}),(0,o.yg)("h1",{id:"the-genome-in-a-bottle-consortium-nist"},"The Genome in a Bottle Consortium (NIST)"),(0,o.yg)("p",null,"We have collected the NIST Ashkenazim (HG003, HG004, HG002) Jewish data set for public use and comparison testing."),(0,o.yg)("h4",{id:"quick-data-access"},"Quick data access:"),(0,o.yg)("ul",null,(0,o.yg)("li",{parentName:"ul"},"Project: ",(0,o.yg)("inlineCode",{parentName:"li"},"A1293-220310-VAR-NIST")),(0,o.yg)("li",{parentName:"ul"},(0,o.yg)("a",{parentName:"li",href:"https://mosaic.chpc.utah.edu/#/projects/922/"},"Mosaic")),(0,o.yg)("li",{parentName:"ul"},(0,o.yg)("a",{parentName:"li",href:"https://uofu.box.com/s/bjap25e6j5pxaqzlvuy1m0xg6b2v9cfh"},"Ubox")),(0,o.yg)("li",{parentName:"ul"},"UCGD Data path: ",(0,o.yg)("inlineCode",{parentName:"li"},"/scratch/ucgd/lustre/UCGD_Datahub/IRBs/proj_UCGDCollab/A1293-220310-VAR-NIST/UCGD"))),(0,o.yg)("h2",{id:"data-collection"},"Data collection."),(0,o.yg)("p",null,"The majority of WGS projects processed within the UCGD are ~30-50x coverage, so a subselection of the ",(0,o.yg)("inlineCode",{parentName:"p"},"Illumina WGS 2x150bp 300X per individual")," data was collected.  To adjust to the desired coverage levels, and based on ",(0,o.yg)("a",{parentName:"p",href:"https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_HiSeq_HG002_Homogeneity-10953946/README_NIST_Illumina_pairedend_HG002.txt"},"this")," NIST README a single data directory was downloaded and processed for each individual, as opposed to downloaded the complete dataset and downsampling."),(0,o.yg)("h3",{id:"important-commands"},"Important commands:"),(0,o.yg)("p",null,(0,o.yg)("strong",{parentName:"p"},"Initially the ",(0,o.yg)("inlineCode",{parentName:"strong"},"140627_D00360_0030_AHA0L6ADXX")," directory was downloaded and processed but it was discovered that this directory had coverage of >1%, so the following directories were used.")),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"Father:\n$> aws s3 sync aws s3 sync s3://giab/data/AshkenazimTrio/HG003_NA24149_father/NIST_HiSeq_HG003_Homogeneity-12389378/HG003_HiSeq300x_fastq/140701_D00360_0032_AHA0KGADXX/ 140701_D00360_0032_AHA0KGADX/\n\nmother:\n$> aws s3 sync s3://giab/data/AshkenazimTrio/HG004_NA24143_mother/NIST_HiSeq_HG004_Homogeneity-14572558/HG004_HiSeq300x_fastq/140818_D00360_0046_AHA5R5ADXX/ 140818_D00360_0046_AHA5R5ADXX\n\nson:\n$> aws s3 sync s3://giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_HiSeq_HG002_Homogeneity-10953946/HG002_HiSeq300x_fastq/140528_D00360_0018_AH8VC6ADXX/ 140528_D00360_0018_AH8VC6ADXX\n\n\nDownloaded size comparison:\n$> du -sch 140701_D00360_0032_AHA0KGADXX/\n69G 140701_D00360_0032_AHA0KGADXX\n\n$> du -sch 140818_D00360_0046_AHA5R5ADXX/\n65G 140818_D00360_0046_AHA5R5ADXX/\n\n$. du -sch 140528_D00360_0018_AH8VC6ADXX/\n67G 140528_D00360_0018_AH8VC6ADXX\n")),(0,o.yg)("h2",{id:"project-setup"},"Project setup"),(0,o.yg)("p",null,"Next the project was set up within UCGD using ",(0,o.yg)("a",{parentName:"p",href:"https://www.nature.com/articles/sdata201625.pdf"},"this")," papers summary information and ",(0,o.yg)("a",{parentName:"p",href:"https://www.coriell.org/1/NIGMS/Collections/NIST-Reference-Materials"},"this")," quick reference."),(0,o.yg)("h2",{id:"comparison-of-version-200-of-the-ucgd-pipeline"},"Comparison of version 2.0.0+ of the UCGD-Pipeline"),(0,o.yg)("p",null,"A Benchmark comparison was run on individual HG002 (UCGD generated file) using the following steps and commands."),(0,o.yg)("h3",{id:"downloaded-benchmark-files"},"Downloaded benchmark files:"),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"$> wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz\n$> wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz.tbi\n$> wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed\n")),(0,o.yg)("h3",{id:"isolated-hg002-individual"},"Isolated HG002 Individual"),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"$> bcftools view -S HG002 A1293-220310-VAR-NIST_Final_1647108329.vcf.gz -O b -o HG002.vcf\n\n## decomposed and sort HG002 individual\n$> vt decompose HG002_UCGD.vcf.gz -o HG002_UCGD.decompose.vcf.gz\n$> bcftools sort HG002_UCGD.decompose.vcf.gz -o HG002_UCGD.decompose.sort.vcf.gz\n$> tabix -p vcf HG002_UCGD.decompose.sort.vcf.gz\n")),(0,o.yg)("h3",{id:"running-rtg"},"Running RTG"),(0,o.yg)("p",null,(0,o.yg)("a",{parentName:"p",href:"https://github.com/RealTimeGenomics/rtg-tools"},"rtg vcfeval")," was used based on 4 different settings:"),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"## (squash-ploidy GQ): Treat heterozygous genotypes as homozygous ALT in both baseline and calls, to allow matches that ignore zygosity differences. \n## all-records: Allow use all records regardless of FILTER status\n\n$> rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --all-records --squash-ploidy -o squash-ploidy -t human_g1k_v38/\n\n## (squash-ploidy-pass) squash-ploidy only allowing 'PASS' calls.\n$> rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --squash-ploidy -o squash-ploidy-pass -t human_g1k_v38/\n\n\n## standard rtg vcfeval run with and without (--all-records)\n## (standard-run)\n$> rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --all-records -o standard-run -t human_g1k_v38/\n## (standard--pass)\n$> rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed -o standard--pass -t human_g1k_v38\n")),(0,o.yg)("h3",{id:"results"},"Results:"),(0,o.yg)("ul",null,(0,o.yg)("li",{parentName:"ul"},"Comparison done on Mar 21 2022")),(0,o.yg)("h4",{id:"roc"},"ROC"),(0,o.yg)("p",null,(0,o.yg)("img",{alt:"ROC",src:n(1787).A,width:"1208",height:"749"})),(0,o.yg)("h4",{id:"precisionsensitivity"},"Precision/Sensitivity"),(0,o.yg)("p",null,(0,o.yg)("img",{alt:"P/S",src:n(5155).A,width:"1213",height:"745"})),(0,o.yg)("h4",{id:"squash-ploidy-pass-summary"},"Squash-ploidy-pass summary:"),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"Overall Summary:\nThreshold  True-pos-baseline  True-pos-call  False-pos  False-neg  Precision  Sensitivity  F-measure\n----------------------------------------------------------------------------------------------------\n    0.000            3516518        3516442       2515     374078     0.9993       0.9039     0.9492\n     None            3587391        3587305      32035     303205     0.9911       0.9221     0.955\n")),(0,o.yg)("h4",{id:"review-of-false-negative-calls"},"Review of False Negative Calls:"),(0,o.yg)("p",null,"To review the possible impact of false negative calls, VEP was ran on the ",(0,o.yg)("inlineCode",{parentName:"p"},"rtg")," produced ",(0,o.yg)("inlineCode",{parentName:"p"},"fn.vcf.gz")," file, and ",(0,o.yg)("a",{parentName:"p",href:"https://vatools.readthedocs.io/en/latest/vep_annotation_reporter.html"},"vep","_","annotation","_","reporter")," was used to collect ClinVar clinical significance annotations."),(0,o.yg)("table",null,(0,o.yg)("thead",{parentName:"table"},(0,o.yg)("tr",{parentName:"thead"},(0,o.yg)("th",{parentName:"tr",align:null},"Count"),(0,o.yg)("th",{parentName:"tr",align:null},"CLIN_SIG"))),(0,o.yg)("tbody",{parentName:"table"},(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"554"),(0,o.yg)("td",{parentName:"tr",align:null},"benign")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"63"),(0,o.yg)("td",{parentName:"tr",align:null},"likely_benign")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"18"),(0,o.yg)("td",{parentName:"tr",align:null},"drug_response")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"11"),(0,o.yg)("td",{parentName:"tr",align:null},"uncertain_significance")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"11"),(0,o.yg)("td",{parentName:"tr",align:null},"protective")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"8"),(0,o.yg)("td",{parentName:"tr",align:null},"benign/likely_benign")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"5"),(0,o.yg)("td",{parentName:"tr",align:null},"pathogenic")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"2"),(0,o.yg)("td",{parentName:"tr",align:null},"conflicting","_","interpretations","_","of","_","pathogenicity")),(0,o.yg)("tr",{parentName:"tbody"},(0,o.yg)("td",{parentName:"tr",align:null},"1"),(0,o.yg)("td",{parentName:"tr",align:null},"risk_factor")))),(0,o.yg)("p",null,"Upon review of the ",(0,o.yg)("inlineCode",{parentName:"p"},"pathogenic")," variants all were found to be co-annotated with the following terms:"),(0,o.yg)("pre",null,(0,o.yg)("code",{parentName:"pre"},"benign\nconflicting_interpretations_of_pathogenicity\nuncertain_significance\n")),(0,o.yg)("h2",{id:"sentieon-dnascope-comparison"},"Sentieon DNAscope comparison."),(0,o.yg)("p",null,"To be added when this comparison is complete."))}g.isMDXComponent=!0},1787:(e,t,n)=>{n.d(t,{A:()=>a});const a=n.p+"assets/images/roc.nist-85304d113063d17e110a893e625bfbca.jpg"},5155:(e,t,n)=>{n.d(t,{A:()=>a});const a=n.p+"assets/images/sensitivity-nist-718d62c91228181cd1c6034da0d90eb7.jpg"}}]);