<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="alternate" type="application/rss+xml" href="/Core-Docs/blog/rss.xml" title="UCGD Core Blog RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/Core-Docs/blog/atom.xml" title="UCGD Core Blog Atom Feed"><title data-react-helmet="true">The Genome in a Bottle Consortium (NIST) | UCGD Core</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://ucgd.github.io/Core-Docs/docs/Public-Datasets/NIST"><meta data-react-helmet="true" name="docusaurus_locale" content="en"><meta data-react-helmet="true" name="docusaurus_version" content="current"><meta data-react-helmet="true" name="docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="The Genome in a Bottle Consortium (NIST) | UCGD Core"><meta data-react-helmet="true" name="description" content="We have collected the NIST Ashkenazim (HG003, HG004, HG002) Jewish data set for public use and comparison testing."><meta data-react-helmet="true" property="og:description" content="We have collected the NIST Ashkenazim (HG003, HG004, HG002) Jewish data set for public use and comparison testing."><link data-react-helmet="true" rel="shortcut icon" href="/Core-Docs/img/ucgd.png"><link data-react-helmet="true" rel="canonical" href="https://ucgd.github.io/Core-Docs/docs/Public-Datasets/NIST"><link data-react-helmet="true" rel="alternate" href="https://ucgd.github.io/Core-Docs/docs/Public-Datasets/NIST" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://ucgd.github.io/Core-Docs/docs/Public-Datasets/NIST" hreflang="x-default"><link rel="stylesheet" href="/Core-Docs/assets/css/styles.04e72905.css">
<link rel="preload" href="/Core-Docs/assets/js/runtime~main.0537e693.js" as="script">
<link rel="preload" href="/Core-Docs/assets/js/main.5304c1b6.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_OuoZ">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a class="navbar__brand" href="/Core-Docs/"><img src="/Core-Docs/img/logo.svg" alt="UCGD Logo" class="themedImage_TMUO themedImage--light_4Vu1 navbar__logo"><img src="/Core-Docs/img/logo.svg" alt="UCGD Logo" class="themedImage_TMUO themedImage--dark_uzRr navbar__logo"><b class="navbar__title">UCGD Core</b></a><a class="navbar__item navbar__link navbar__link--active" href="/Core-Docs/docs/Homepage">Documentation</a></div><div class="navbar__items navbar__items--right"><a href="https://github.com/UCGD/Core-Docs" target="_blank" rel="noopener noreferrer" class="navbar__item navbar__link"><span>GitHub<svg width="13.5" height="13.5" aria-hidden="true" viewBox="0 0 24 24" class="iconExternalLink_wgqa"><path fill="currentColor" d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"></path></svg></span></a><div class="react-toggle toggle_2i4l react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_iYfV">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_iYfV">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_lDyR"><button class="clean-btn backToTopButton_i9tI" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_0YBq"><div class="sidebar_a3j0"><nav class="menu thin-scrollbar menu_cyFh menuWithAnnouncementBar_+O1J"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/Core-Docs/docs/Homepage">Homepage</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">CHPC_Guide</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Public-Datasets</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/Core-Docs/docs/Public-Datasets/NIST">The Genome in a Bottle Consortium (NIST)</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">UCGD-Pipeline</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">UCGD-Projects</a></li></ul></nav></div></aside><main class="docMainContainer_r8cw"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_zHA2"><div class="docItemContainer_oiyr"><article><div class="tocCollapsible_aw-L theme-doc-toc-mobile tocMobile_Tx6Y"><button type="button" class="clean-btn tocCollapsibleButton_zr6a">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>The Genome in a Bottle Consortium (NIST)</h1></header><p>We have collected the NIST Ashkenazim (HG003, HG004, HG002) Jewish data set for public use and comparison testing.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_y2LR" id="quick-data-access"></a>Quick data access:<a class="hash-link" href="#quick-data-access" title="Direct link to heading">#</a></h4><ul><li>Project: <code>A1293-220310-VAR-NIST</code></li><li><a href="https://mosaic.chpc.utah.edu/#/projects/922/" target="_blank" rel="noopener noreferrer">Mosaic</a></li><li><a href="https://uofu.box.com/s/bjap25e6j5pxaqzlvuy1m0xg6b2v9cfh" target="_blank" rel="noopener noreferrer">Ubox</a></li><li>UCGD Data path: <code>/scratch/ucgd/lustre/UCGD_Datahub/IRBs/proj_UCGDCollab/A1293-220310-VAR-NIST/UCGD</code></li></ul><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_y2LR" id="data-collection"></a>Data collection.<a class="hash-link" href="#data-collection" title="Direct link to heading">#</a></h2><p>The majority of WGS projects processed within the UCGD are ~30-50x coverage, so a subselection of the <code>Illumina WGS 2x150bp 300X per individual</code> data was collected.  To adjust to the desired coverage levels, and based on <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_HiSeq_HG002_Homogeneity-10953946/README_NIST_Illumina_pairedend_HG002.txt" target="_blank" rel="noopener noreferrer">this</a> NIST README a single data directory was downloaded and processed for each individual, as opposed to downloaded the complete dataset and downsampling.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_y2LR" id="important-commands"></a>Important commands:<a class="hash-link" href="#important-commands" title="Direct link to heading">#</a></h3><p><strong>Initially the <code>140627_D00360_0030_AHA0L6ADXX</code> directory was downloaded and processed but it was discovered that this directory had coverage of &gt;1%, so the following directories were used.</strong></p><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">Father:</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; aws s3 sync aws s3 sync s3://giab/data/AshkenazimTrio/HG003_NA24149_father/NIST_HiSeq_HG003_Homogeneity-12389378/HG003_HiSeq300x_fastq/140701_D00360_0032_AHA0KGADXX/ 140701_D00360_0032_AHA0KGADX/</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">mother:</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; aws s3 sync s3://giab/data/AshkenazimTrio/HG004_NA24143_mother/NIST_HiSeq_HG004_Homogeneity-14572558/HG004_HiSeq300x_fastq/140818_D00360_0046_AHA5R5ADXX/ 140818_D00360_0046_AHA5R5ADXX</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">son:</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; aws s3 sync s3://giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_HiSeq_HG002_Homogeneity-10953946/HG002_HiSeq300x_fastq/140528_D00360_0018_AH8VC6ADXX/ 140528_D00360_0018_AH8VC6ADXX</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">Downloaded size comparison:</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; du -sch 140701_D00360_0032_AHA0KGADXX/</span></span><span class="token-line" style="color:#393A34"><span class="token plain">69G 140701_D00360_0032_AHA0KGADXX</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; du -sch 140818_D00360_0046_AHA5R5ADXX/</span></span><span class="token-line" style="color:#393A34"><span class="token plain">65G 140818_D00360_0046_AHA5R5ADXX/</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$. du -sch 140528_D00360_0018_AH8VC6ADXX/</span></span><span class="token-line" style="color:#393A34"><span class="token plain">67G 140528_D00360_0018_AH8VC6ADXX</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_y2LR" id="project-setup"></a>Project setup<a class="hash-link" href="#project-setup" title="Direct link to heading">#</a></h2><p>Next the project was set up within UCGD using <a href="https://www.nature.com/articles/sdata201625.pdf" target="_blank" rel="noopener noreferrer">this</a> papers summary information and <a href="https://www.coriell.org/1/NIGMS/Collections/NIST-Reference-Materials" target="_blank" rel="noopener noreferrer">this</a> quick reference.</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_y2LR" id="comparison-of-version-200-of-the-ucgd-pipeline"></a>Comparison of version 2.0.0+ of the UCGD-Pipeline<a class="hash-link" href="#comparison-of-version-200-of-the-ucgd-pipeline" title="Direct link to heading">#</a></h2><p>A Benchmark comparison was run on individual HG002 (UCGD generated file) using the following steps and commands.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_y2LR" id="downloaded-benchmark-files"></a>Downloaded benchmark files:<a class="hash-link" href="#downloaded-benchmark-files" title="Direct link to heading">#</a></h3><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz.tbi</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; wget https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/AshkenazimTrio/HG002_NA24385_son/latest/GRCh38/HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_y2LR" id="isolated-hg002-individual"></a>Isolated HG002 Individual<a class="hash-link" href="#isolated-hg002-individual" title="Direct link to heading">#</a></h3><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; bcftools view -S HG002 A1293-220310-VAR-NIST_Final_1647108329.vcf.gz -O b -o HG002.vcf</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">## decomposed and sort HG002 individual</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; vt decompose HG002_UCGD.vcf.gz -o HG002_UCGD.decompose.vcf.gz</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; bcftools sort HG002_UCGD.decompose.vcf.gz -o HG002_UCGD.decompose.sort.vcf.gz</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; tabix -p vcf HG002_UCGD.decompose.sort.vcf.gz</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_y2LR" id="running-rtg"></a>Running RTG<a class="hash-link" href="#running-rtg" title="Direct link to heading">#</a></h3><p><a href="https://github.com/RealTimeGenomics/rtg-tools" target="_blank" rel="noopener noreferrer">rtg vcfeval</a> was used based on 4 different settings:</p><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">## (squash-ploidy GQ): Treat heterozygous genotypes as homozygous ALT in both baseline and calls, to allow matches that ignore zygosity differences. </span></span><span class="token-line" style="color:#393A34"><span class="token plain">## all-records: Allow use all records regardless of FILTER status</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --all-records --squash-ploidy -o squash-ploidy -t human_g1k_v38/</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">## (squash-ploidy-pass) squash-ploidy only allowing &#x27;PASS&#x27; calls.</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --squash-ploidy -o squash-ploidy-pass -t human_g1k_v38/</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain" style="display:inline-block">
</span></span><span class="token-line" style="color:#393A34"><span class="token plain">## standard rtg vcfeval run with and without (--all-records)</span></span><span class="token-line" style="color:#393A34"><span class="token plain">## (standard-run)</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed --all-records -o standard-run -t human_g1k_v38/</span></span><span class="token-line" style="color:#393A34"><span class="token plain">## (standard--pass)</span></span><span class="token-line" style="color:#393A34"><span class="token plain">$&gt; rtg-tools-3.12.1/rtg vcfeval -b HG002_GRCh38_1_22_v4.2.1_benchmark.sort.vcf.gz -c HG002_UCGD.decompose.sort.vcf.gz --bed-regions HG002_GRCh38_1_22_v4.2.1_benchmark_noinconsistent.bed -o standard--pass -t human_g1k_v38</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_y2LR" id="results"></a>Results:<a class="hash-link" href="#results" title="Direct link to heading">#</a></h3><ul><li>Comparison done on Mar 21 2022</li></ul><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_y2LR" id="roc"></a>ROC<a class="hash-link" href="#roc" title="Direct link to heading">#</a></h4><p><img alt="ROC" src="/Core-Docs/assets/images/roc.nist-85304d113063d17e110a893e625bfbca.jpg"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_y2LR" id="precisionsensitivity"></a>Precision/Sensitivity<a class="hash-link" href="#precisionsensitivity" title="Direct link to heading">#</a></h4><p><img alt="P/S" src="/Core-Docs/assets/images/sensitivity-nist-718d62c91228181cd1c6034da0d90eb7.jpg"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_y2LR" id="squash-ploidy-pass-summary"></a>Squash-ploidy-pass summary:<a class="hash-link" href="#squash-ploidy-pass-summary" title="Direct link to heading">#</a></h4><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">Overall Summary:</span></span><span class="token-line" style="color:#393A34"><span class="token plain">Threshold  True-pos-baseline  True-pos-call  False-pos  False-neg  Precision  Sensitivity  F-measure</span></span><span class="token-line" style="color:#393A34"><span class="token plain">----------------------------------------------------------------------------------------------------</span></span><span class="token-line" style="color:#393A34"><span class="token plain">    0.000            3516518        3516442       2515     374078     0.9993       0.9039     0.9492</span></span><span class="token-line" style="color:#393A34"><span class="token plain">     None            3587391        3587305      32035     303205     0.9911       0.9221     0.955</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_y2LR" id="review-of-false-negative-calls"></a>Review of False Negative Calls:<a class="hash-link" href="#review-of-false-negative-calls" title="Direct link to heading">#</a></h4><p>To review the possible impact of false negative calls, VEP was ran on the <code>rtg</code> produced <code>fn.vcf.gz</code> file, and <a href="https://vatools.readthedocs.io/en/latest/vep_annotation_reporter.html" target="_blank" rel="noopener noreferrer">vep_annotation_reporter</a> was used to collect ClinVar clinical significance annotations.</p><table><thead><tr><th>Count</th><th>CLIN_SIG</th></tr></thead><tbody><tr><td>554</td><td>benign</td></tr><tr><td>63</td><td>likely_benign</td></tr><tr><td>18</td><td>drug_response</td></tr><tr><td>11</td><td>uncertain_significance</td></tr><tr><td>11</td><td>protective</td></tr><tr><td>8</td><td>benign/likely_benign</td></tr><tr><td>5</td><td>pathogenic</td></tr><tr><td>2</td><td>conflicting_interpretations_of_pathogenicity</td></tr><tr><td>1</td><td>risk_factor</td></tr></tbody></table><p>Upon review of the <code>pathogenic</code> variants all were found to be co-annotated with the following terms:</p><div class="codeBlockContainer_J+bg"><div class="codeBlockContent_csEI"><pre tabindex="0" class="prism-code language-undefined codeBlock_rtdJ thin-scrollbar" style="color:#393A34;background-color:#f6f8fa"><code class="codeBlockLines_1zSZ"><span class="token-line" style="color:#393A34"><span class="token plain">benign</span></span><span class="token-line" style="color:#393A34"><span class="token plain">conflicting_interpretations_of_pathogenicity</span></span><span class="token-line" style="color:#393A34"><span class="token plain">uncertain_significance</span></span></code></pre><button type="button" aria-label="Copy code to clipboard" class="copyButton_M3SB clean-btn">Copy</button></div></div><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_y2LR" id="sentieon-dnascope-comparison"></a>Sentieon DNAscope comparison.<a class="hash-link" href="#sentieon-dnascope-comparison" title="Direct link to heading">#</a></h2><p>To be added when this comparison is complete.</p></div><footer class="theme-doc-footer docusaurus-mt-lg"><div class="theme-doc-footer-edit-meta-row row"><div class="col"><a href="https://github.com/UCGD/Core-Docs/docs/docs/Public-Datasets/NIST.md" target="_blank" rel="noreferrer noopener" class="theme-edit-this-page"><svg fill="currentColor" height="20" width="20" viewBox="0 0 40 40" class="iconEdit_mS5F" aria-hidden="true"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col lastUpdated_mt2f"></div></div></footer></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/Core-Docs/docs/CHPC_Guide/iTerm_Configuration"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« iTerm Configuration</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/Core-Docs/docs/UCGD-Pipeline/additional_software"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Adding software Â»</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_vrFS thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#data-collection" class="table-of-contents__link">Data collection.</a><ul><li><a href="#important-commands" class="table-of-contents__link">Important commands:</a></li></ul></li><li><a href="#project-setup" class="table-of-contents__link">Project setup</a></li><li><a href="#comparison-of-version-200-of-the-ucgd-pipeline" class="table-of-contents__link">Comparison of version 2.0.0+ of the UCGD-Pipeline</a><ul><li><a href="#downloaded-benchmark-files" class="table-of-contents__link">Downloaded benchmark files:</a></li><li><a href="#isolated-hg002-individual" class="table-of-contents__link">Isolated HG002 Individual</a></li><li><a href="#running-rtg" class="table-of-contents__link">Running RTG</a></li><li><a href="#results" class="table-of-contents__link">Results:</a></li></ul></li><li><a href="#sentieon-dnascope-comparison" class="table-of-contents__link">Sentieon DNAscope comparison.</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="row footer__links"><div class="col footer__col"><div class="footer__title">UCGD Docs</div><ul class="footer__items"><li class="footer__item"><a class="footer__link-item" href="/Core-Docs/docs/Homepage">Documentation</a></li></ul></div><div class="col footer__col"><div class="footer__title">UCGD Links</div><ul class="footer__items"><li class="footer__item"><a href="https://github.com/UCGD" target="_blank" rel="noopener noreferrer" class="footer__link-item"><span>UCGD Github page<svg width="13.5" height="13.5" aria-hidden="true" viewBox="0 0 24 24" class="iconExternalLink_wgqa"><path fill="currentColor" d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"></path></svg></span></a></li><li class="footer__item"><a href="https://cores.utah.edu/ucgd/" target="_blank" rel="noopener noreferrer" class="footer__link-item"><span>UCGD Cores Page<svg width="13.5" height="13.5" aria-hidden="true" viewBox="0 0 24 24" class="iconExternalLink_wgqa"><path fill="currentColor" d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"></path></svg></span></a></li></ul></div><div class="col footer__col"><div class="footer__title">More</div><ul class="footer__items"><li class="footer__item"><a href="https://github.com/UCGD/Core-Docs" target="_blank" rel="noopener noreferrer" class="footer__link-item"><span>Documentation GitHub Page<svg width="13.5" height="13.5" aria-hidden="true" viewBox="0 0 24 24" class="iconExternalLink_wgqa"><path fill="currentColor" d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"></path></svg></span></a></li></ul></div></div><div class="footer__bottom text--center"></div></div></footer></div>
<script src="/Core-Docs/assets/js/runtime~main.0537e693.js"></script>
<script src="/Core-Docs/assets/js/main.5304c1b6.js"></script>
</body>
</html>